README.md

```diff
@@ -97,6 +115,20 @@ The Information Assistant (IA) Accelerator and Microsoft are committed to the ad
 
 Find out more with Microsoft's [Responsible AI resources](https://www.microsoft.com/en-us/ai/responsible-ai)
 
+### Content Safety
+
+Content safety is provided through Azure Open AI service.The Azure OpenAI Service includes a content filtering system that runs alongside the core AI models. This system uses an ensemble of classification models to detect four categories of potentially harmful content (violence, hate, sexual, and self-harm) at four severity levels (safe, low, medium, high).
+
+By default, the content filters are set to filter out prompts and completions that are detected as medium or high severity for those four harm categories. Content labeled as low or safe severity is not filtered.
```

Comment:
Should it be mentioned that those 4 categories may not be sufficient for all use cases, especially for minors, and have a link to the Transparency Doc?

Reply:
fixed